src/Components/Facility/models.tsx

```diff
@@ -222,6 +227,230 @@ export interface CurrentBed {
+  | "output";
+
+export type NutritionPlotsRes = {
+  infusions: [any];
+  iv_fluids: [any];
+  feeds: [any];
+  total_intake_calculated: string;
+  total_output_calculated: string;
+  output: [any];
+};
+
+export type PainDiagramsFields = "pain_scale_enhanced";
+
+export type PainDiagramsRes = {
+  pain_scale_enhanced: [any];
+};
+
+export type PressureSoreDiagramsFields = "pressure_sore";
+
+export type PressureSoreDiagramsRes = {
+  pressure_sore: [any];
+};
+
+export type PrimaryParametersPlotFields =
+  | "bp"
+  | "pulse"
+  | "temperature"
+  | "resp"
+  | "blood_sugar_level"
+  | "insulin_intake_frequency"
+  | "insulin_intake_dose"
+  | "ventilator_spo2"
+  | "ventilator_fi02"
+  | "rhythm"
+  | "rhythm_detail";
+
+export type PrimaryParametersPlotRes = {
+  bp: {
+    mean?: number;
+    systolic?: number;
+    diastolic?: number;
+  };
+  pulse: number;
+  temperature: string;
+  resp: number;
+  blood_sugar_level: number;
+  insulin_intake_frequency: number;
+  insulin_intake_dose: string;
+  ventilator_spo2: number;
+  ventilator_fi02: number;
+  rhythm: number;
+  rhythm_detail: string;
+};
+
+export type VentilatorPlotFields =
+  | "ventilator_pip"
+  | "ventilator_mean_airway_pressure"
+  | "ventilator_resp_rate"
+  | "ventilator_pressure_support"
+  | "ventilator_tidal_volume"
+  | "ventilator_peep"
+  | "ventilator_fi02"
+  | "ventilator_spo2"
+  | "etco2"
+  | "bilateral_air_entry"
+  | "ventilator_oxygen_modality_oxygen_rate"
+  | "ventilator_oxygen_modality_flow_rate";
+
+export type VentilatorPlotRes = {
+  ventilator_pip: number;
+  ventilator_mean_airway_pressure: number;
+  ventilator_resp_rate: number;
+  ventilator_pressure_support: number;
+  ventilator_tidal_volume: number;
+  ventilator_peep: string;
+  ventilator_fi02: number;
+  ventilator_spo2: number;
+  etco2: number;
+  bilateral_air_entry: boolean;
+  ventilator_oxygen_modality_oxygen_rate: number;
+  ventilator_oxygen_modality_flow_rate: number;
+};
+
+export interface DailyRoundsBody {
+  page?: number;
+  fields:
+    | ABGPlotsFields[]
+    | DialysisPlotsFields[]
+    | NeurologicalTablesFields[]
+    | NursingPlotFields[]
+    | NutritionPlotsFields[]
+    | PainDiagramsFields[]
+    | PressureSoreDiagramsFields[]
+    | PrimaryParametersPlotFields[]
+    | VentilatorPlotFields[];
+}
+
+export interface DailyRoundsRes {
+  count: number;
+  page_size: number;
+  results:
+    | ABGPlotsRes
+    | DialysisPlotsRes
+    | NeurologicalTablesRes
+    | NursingPlotRes
+    | NutritionPlotsRes
+    | PainDiagramsRes
+    | PressureSoreDiagramsRes
+    | PrimaryParametersPlotRes
+    | VentilatorPlotRes;
+}
+
+export interface GetBedsRes {
+  count: number;
+  results: CurrentBed[];
+}
+
+export interface CreateBedBody {
+  start_date: string;
+  assets: string[];
+  consultation: string;
+  bed: string;
+}
+
+export interface GetDailyReportsBody {
+  limit: number;
+  offset: number;
+  rounds_type: "" | "NORMAL,VENTILATOR,ICU";
+}
```

Comment:
You don't need to define types for path params. GET requests do not have a request body.